Resources/Private/Templates/Backend/Guest.html

```diff
@@ -1,6 +1,6 @@
 <f:section name="guestFrameApplication">
     <script>window.neos = window.parent.neos;</script>
     <script src="{f:uri.resource(path: 'resource://Neos.Neos.Ui/Public/Plugins/ckeditor/ckeditor.js')}"></script>
-    <script src="{f:uri.resource(path: 'resource://Neos.Neos.Ui/Public/JavaScript/Guest.js')}"></script>
-    <link rel="stylesheet" href="{f:uri.resource(path: 'resource://Neos.Neos.Ui/Public/Styles/Guest.css')}">
+    <script src="{f:uri.resource(path: 'resource://Neos.Neos.Ui/Public/Javascript/Guest.js')}"></script>
+    <link rel="stylesheet" href="{f:uri.resource(path: 'resource://Neos.Neos.Ui/Public/Styles/Host.css')}">
```

Comment:
Hey, I agree but I do not know how we could do that, as the file is currently called "Host.js" and "Host.css". I don't know how we can change the naming for CSS, but not for JS...